packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.23
+
+* **BREAKING CHANGE:** StoreKit 2 is now the default for all devices that support it.
```

Comment:
should this be 0.4.0 instead? iirc breaking change should be major bump, but this is still 0. version, so a minor bump?
(cc @stuartmorgan-g for advice)

Reply:
Yes, breaking changes need a "major" (which for pre-1.0 is the second number, per Dart convention) version bump.

Reply:
ooh makes sense. Ive changed it.